File: Competitive Programming 3/Section 1.3.3/12577 - Hajj-e-Akbar/C++/main.cpp
#include <iostream>

using namespace std;

string line;

int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);
    for(int cs=1;getline(cin,line)&&line[0]!='*';cs++){
        cout << "Case " << cs  << ": Hajj-e-" << (line[0]=='U'?"Asghar":"Akbar") << "\n";
    }
    return 0;
}

File: Competitive Programming 3/Section 1.3.3/12577 - Hajj-e-Akbar/Java/Main.java
import java.util.Scanner;

public class Main {
    static Scanner s = new Scanner(System.in);
    public static void main(String[] args){
        String line ; 
        for(int cs = 1;;cs++){
            String ans = "Akbar";
            line = s.next();
            if(line.charAt(0)=='U'){
                ans = "Asghar";
            }else if(line.charAt(0)=='*'){
                break;
            }
            System.out.printf("Case %d: %s\n",cs,"Hajj-e-"+ans);
        }
    }
}